test/adapters/parsers/csv.spec.js

```diff
@@ -15,7 +15,7 @@ describe('parsers/csv', function() {
 
     it('fails when given an invalid CSV stream', function() {
         var csv = parsers.getParser('csv');
-        csv.parseStream(fs.createReadStream(invalidFile))
+        return csv.parseStream(fs.createReadStream(invalidFile))
```

Comment:
add the `function() {}` in here for emit since its generating those failures due to the timing of error out vs hitting the emit in the `end` event.